runtime/stateManager.py

```diff
@@ -256,12 +301,19 @@ def hibike_read_params(self, pipe, uid, params):
         pipe.send([HIBIKE_COMMANDS.READ.value, [uid, params]])
 
     def hibike_timestamp_down(self, pipe, *data):
-        print("We have received timestamp_down")
+        """
+        Receives timestamp from HBIKE; sends the data including timestamp
+         down to the queue.
+        """
         data = list(data)
         data.append(time.perf_counter())
         pipe.send([HIBIKE_COMMANDS.TIMESTAMP_DOWN.value, data])
 
     def hibike_response_device_subbed(self, uid, delay, params):
+        """
+        Subscribes to recognized devices; creates a entry in STATE
```

Comment:
Again, a bit too literal. This function doesn't actually subscribe to anything, it just acknowledges that a subscription has occurred.